bootstrap/etc/salt/cloud.maps.d/salt_masters.yml

```diff
@@ -0,0 +1,32 @@
+# -*- mode: yaml; coding: utf-8; -*-
+salt_master:
+  - master-operations-production:
+      grains:
+        roles:
+          - master
+          - master-production
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: True
+          SubnetId: sdb://osenv/PROD_SUBNET
+          SecurityGroupId: sdb://osenv/SECURITY_GROUPS
+      tag:
+        environment: operations
+        role: master-production
+        OU: operations
+        Environment: operations
+  - master-operations-qa:
+      grains:
+        roles:
+          - master
+          - master-qa
+      network_interfaces:
+        - DeviceIndex: 0
+          AssociatePublicIpAddress: True
+          SubnetId: sdb://osenv/QA_SUBNET
+          SecurityGroupId: sdb://osenv/SECURITY_GROUPS
```

Comment:
Should this have a different value from the production one?

Reply:
This file is actually going to be deleted in the next commit. It was a failed effort that got moved into the orchestrate state.